src/export-transactions.ts

```diff
@@ -82,19 +83,27 @@ const BALANCES = (RAW_BALANCES as BalanceParsed[]).reduce((accum, balance) => {
 
 async function getTopicTxs(network: Network, startblock: number, topic: Topic) {
   let block = startblock
+  const MAX_BLOCK_RANGE = 1000000
   const response = await fetchCovalentURL<{ height: number }>(`${baseCovalentUrl(network)}/block_v2/latest/?key=${COVALENT_API_KEY}`, 0)
   const latestBlock = response[0].height
   console.log(`Latest ${JSON.stringify(network)} - start block: ${block} - latest block: ${latestBlock}`)
+  const unresolvedEvents: Promise<EventItem[]>[] = []
+  
+  while(block < latestBlock) {
+    const endBlock = block + MAX_BLOCK_RANGE
+    const url = `${baseCovalentUrl(network)}/events/topics/${topic}/?key=${COVALENT_API_KEY}&starting-block=${block}&ending-block=${endBlock > latestBlock ? latestBlock : endBlock}`
+    console.log('fetch', url)
```

Comment:
Do we want to log this?

Reply:
Yes, it's useful to have the full url in case the fetch fails for some reason to test it!

Reply:
Maybe a description about what the URL is about instead of "fetch"?